docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,285 @@
+          - if the draw pattern is HIGH, the expected UEF is 0.92, for draw all draw patterns other draw patterns, it is 0.91: `if draw_pattern == "HIGH": expected_UEF = 0.92; else: expected_UEF = 0.91`
+          - check if the actual uniform_energy_factor is >= to the expected: `if shw_eq.uniform_energy_factor >= expected_UEF:`
+            - set rule_status to "PASS": `rule_status = "PASS"`
+          - otherwise, set rule status to FAIL and provide the note that the expected UEF is less than the required UEF: `else: rule_status = "FAIL"; rule_note = "The modeled Uniform energy Factor (UEF - " + shw_eq.uniform_energy_factor + ") is less than required (" + expected_UEF + ")"`
+        - else, if the storage tank is is between 20 and 55 gallons, this is considered an electric storage water heater in Appendix F: `elsif storage_volume >= 20 && storage_volume <= 55:`
+          - determine the UEF based on the draw pattern for VERY_SMALL: `if draw_pattern == "VERY_SMALL": expected_uef = 0.8808 – (0.0008 × storage_volume)`
+          - determine the UEF based on the draw pattern for LOW: `if draw_pattern == "LOW": expected_uef = 0.9254 – (0.0003 × storage_volume)`
+          - determine the UEF based on the draw pattern for MEDIUM: `if draw_pattern == "MEDIUM": expected_uef = 0.9307 – (0.0002 × storage_volume)`
+          - determine the UEF based on the draw pattern for HIGH: `if draw_pattern == "HIGH": expected_uef = 0.9349 – (0.0001 × storage_volume)` 
+          - check if the actual uniform_energy_factor is >= to the expected: `if shw_eq.uniform_energy_factor >= expected_UEF:`
+            - set rule_status to "PASS": `rule_status = "PASS"`
+          - otherwise, set rule status to FAIL and provide the note that the expected UEF is less than the required UEF: `else: rule_status = "FAIL"; rule_note = "The modeled Uniform energy Factor (UEF - " + shw_eq.uniform_energy_factor + ") is less than required (" + expected_UEF + ")"`
+        - else, if the storage tank is greater than 55 and less than 100 gallons, Appendix F provides a heat pump efficiency, or the storage volume is between 2 and 20 gallons, in which case there is no rating in APPENDIX F.  return UNDETERMINED: `else:`
+          - set rule_status to UNDETERMINED: `rule_status = "UNDETERMINED"`
+      - otherwise, continue with rule logic: `else:`
+        - check electric storage water heaters with capacity greater than 12 kW: `if type == "ELECTRIC_RESISTANCE_STORAGE" && input_capacity > 40945.7:`
+          - check that the capacity_per_volume < 4000 btu/hr/gallon: `if capacity_per_volume < 4000:`
+            - calculate the standby loss target: `standby_loss_target = 0.3 + (27/swh_eq.distribution_system.tank.storage_volume)` 
+            - if the standby_loss is less than or equal to the target, set the rule_status to PASS: `if standby_loss <= standby_loss_target: rule_status: "PASS"`
```

Comment:
we should probbaly have a standby loss data element rather than the standy_loss_fraction since the standby_loss_fraction appear specifci to CA title 24